config/routes.rb

```diff
@@ -100,6 +100,8 @@
     get 'terms_and_conditions/:name/versions/latest', to: 'terms_and_conditions#latest'
     get 'terms_and_conditions/:name/versions/latest/user_data', to: 'terms_and_conditions#latest_user_data'
     post 'terms_and_conditions/:name/versions/latest/user_data', to: 'terms_and_conditions#accept_latest'
+
+    resource :health_beta_registrations, only: [:show, :create]
```

Comment:
Need to make sure this is behind HTTP basic auth, possibly with something other than what we typically use (not mentioning it here because this is public repo).